xapian-core/include/xapian/weight.h

```diff
@@ -409,6 +409,11 @@ class XAPIAN_VISIBILITY_DEFAULT TfIdfWeight : public Weight {
     /// The factor to multiply with the weight.
     double factor;
 
+    /// Parameters slope and delta in the Piv+ normalization weighting formula.
+    /// Default values: 0.2 and 1.0 respectively.
+    double param_slope = 0.2;
+    double param_delta = 1.0;
+
     TfIdfWeight * clone() const;
```

Comment:
We've so far avoided C++11 features in the public API headers (because many commonly used compiler versions need C++11 enabling explicitly, and if we use C++11 features, projects using Xapian will need to make their build systems detect what (if any) option is needed.

Given this one is easy to avoid, I think we should - just initialise in the default constructor instead.  It's probably clearer to keep the initialisations together anyway.